pkg/cdi/cache.go

```diff
@@ -69,7 +69,9 @@ func NewCache(options ...Option) (*Cache, error) {
 		watch:       &watch{},
 	}
 
-	WithSpecDirs(DefaultSpecDirs...)(c)
+	if err := WithSpecDirs(DefaultSpecDirs...)(c); err != nil {
```

Comment:
WithSpecDirs can't fail, so this is fine.

Reply:
Hmm, ok, this lead me to submit #187 (replaces this particular change)